Cargo.toml

```diff
@@ -83,35 +56,12 @@ felt = { package = "cairo-felt", path = "./felt", version = "0.5.1", default-fea
 bitvec = { version = "1", default-features = false, features = ["alloc"] }
 
 # Dependencies for cairo-1-hints feature
-cairo-lang-starknet = {version = "1.1.0", optional = true}
-cairo-lang-casm = {version = "1.1.0", optional = true }
+cairo-lang-starknet = { version = "1.1.0"}
+cairo-lang-casm = { version = "1.1.0"}
```

Comment:
Why are these no longer optional?

Reply:
The workspace does not accept optional dependencies.
It is just a register of versions of crates for the crates `Cargo.toml` to use. 
The crate's responsible for saying if they are optional or not.